doc/api/sessions/valid_params.html

```diff
@@ -139,11 +139,11 @@ <h4>Headers</h4>
 X-Permitted-Cross-Domain-Policies: none
 Referrer-Policy: strict-origin-when-cross-origin
 Content-Type: application/json; charset=utf-8
-ETag: W/&quot;d1396303703d3d3d1385a985b8910a30&quot;
+ETag: W/&quot;7d592b1ea9476e5a9162458880bab98c&quot;
 Cache-Control: max-age=0, private, must-revalidate
-X-Request-Id: 49760746-f04f-4351-8ff8-ebd96c0d7fbb
-X-Runtime: 0.251310
-Content-Length: 273</pre>
+X-Request-Id: 36c34f57-d16c-4454-8849-5148100dcd20
+X-Runtime: 0.255051
+Content-Length: 272</pre>
```

Comment:
What are these changes about?

Reply:
This file is autogenerated documentation. I will rebase this branch in order to discard this file. @sebastiancaraballo